lib/ansible/modules/system/cgroup.py

```diff
@@ -0,0 +1,297 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2018, Cédric Nisio <cedric.nisio@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+module: cgroup
+
+short_description: Manage cgroup configurations
+
+version_added: "2.6"
+
+description:
+    - Manages cgroup configurations in the C(/etc/cgconfig.d) directory.
+
+options:
+    name:
+        description:
+            - The cgroup name.
+        required: true
+    state:
+        description:
+            - Whether the cgroup should be present or absent.
+        required: true
```

Comment:
by convention the state should be defaulted to `present` whenever possible, as a result th

~~~diff
-        required: true
+        default: present
~~~

Reply:
Done